app/main.py

```diff
@@ -1,3 +1,60 @@
-def shop_trip():
-    # write your code here
-    pass
+from app.utils.utils import (
+    file_handler,
+    fuel_cost_one_way,
+    trip_cost
+)
+from app.instances.customer import Customer
+from app.instances.shop import Shop
+from app.instances.car import Car
+
+
+def shop_trip() -> None:
+    config_data, fuel_price = file_handler("app/config.json")
+
+    customers = []
+    for customer in config_data["customers"]:
+        customers.append(
+            Customer
+            (
+                customer["name"],
+                customer["product_cart"],
+                customer["location"],
+                customer["money"],
+                Car(customer["car"]["brand"],
+                    customer["car"]["fuel_consumption"])
+            )
+        )
```

Comment:
You can use `**` to unpack and list comprehenstion to create instances. Mind everywhere

Reply:
when I try to unpack and do list comprehension (def init_customers(customers_data: dict) -> List[Customer]:
    return [
        Customer(**customer, car=Car(**customer["car"]))
        for customer in customers_data["customers"]
    ])
    I get error like this - TypeError: app.instances.customer.Customer() got multiple values for keyword argument 'car'
    Can you suggest how this can be fixed?